shortlister/model.py

```diff
@@ -83,3 +83,6 @@ def load_criteria(csv_file):
             criterion = Criterion(name=row[0],description=row[1],scores= tuple(row[2].split(",")))
             criteria.append(criterion)
     return criteria
+
+def update_applicant_score(applicant_score,criterion,score_index):
```

Comment:
I would change the signature of this function to `(applicant: Applicant, criterion: Criterion, score_index: int)`. That hides the internal storage of applicant information from the controller.